.github/workflows-src/build-and-upload-for-pypi-pure.yml

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2017-2021  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+
+name: "Build (pure python) packages and publish on PyPI"
+
+on:
+  push:
```

Comment:
shouldn't we limit CI to run on pushes to `master` only?

Reply:
It only pushes to PyPI on `master`, the other ones it just builds the files and uploads it as artifacts.